src/main/kotlin/lotto/domain/selling/Rank.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domain.selling
+
+enum class Rank(val matchCount: Int, val prizeMoney: Int) {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    MISS(0, 0);
+
+    companion object {
+        private val ranks: HashMap<Int, Rank> = HashMap(values().associateBy { it.matchCount })
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        private val ranks: Map<Int, Rank> = values().associateBy { it.matchCount }
```